src/modules/vax/VaxLocationDetail.tsx

```diff
@@ -153,6 +174,106 @@ export default function VaxLocationDetail({ loading, isUserLocationExist, locati
+            </Text>
+            <Text as="h3" fontSize="sm" fontWeight={600} textTransform="capitalize">
+              {wilayah.toLowerCase()}
+            </Text>
+            <Link
+              _focus={{
+                outline: 'none'
+              }}
+              _hover={{
+                color: 'blue.500'
+              }}
+              color="blue.300"
+              fontWeight="semibold"
+              href={mapsUrl}
+              isExternal
+            >
+              <Text fontSize="md">
+                📍 Cari Lokasi
+                <ExternalLinkIcon aria-hidden mx={2} />
+              </Text>
+            </Link>
+          </DrawerHeader>
+          <DrawerBody>
+            <Box mb={2}>
+              <Text as="h3" fontSize="lg" fontWeight={700}>
+                💉 Jadwal Vaksinasi{' '}
+                <Badge borderRadius="10">
+                  <Text
+                    as="span"
+                    color={colorMode === 'dark' ? 'gray.300' : 'gray.600'}
+                    fontSize="xs"
+                    fontWeight={400}
+                    gridArea="timestamp"
+                    isTruncated
+                  >
+                    <Text as="time" dateTime={new Date(lastUpdated).toISOString()}>
+                      {formatDistanceToNow(Date.parse(lastUpdated), { locale: idLocale, addSuffix: true })}
+                    </Text>
+                  </Text>
+                </Badge>
+              </Text>
+            </Box>
+            <Divider mb={2} />
+            <Stack direction="column" spacing={8}>
+              {jadwal.map(({ id: jadwalId, waktu }) => (
+                <Box key={jadwalId}>
+                  <Text as="h3" color="gray.500" fontSize="sm" fontWeight={600} mb={2}>
+                    {format(parse(jadwalId, 'yyyy-MM-dd', new Date()), 'PPPP', { locale: idLocale })}
+                  </Text>
+                  <Table size="sm">
+                    <Thead>
+                      <Tr>
+                        <Th>Waktu</Th>
+                        <Th>Sisa Kuota</Th>
+                        <Th>Total Kuota</Th>
+                      </Tr>
+                    </Thead>
+                    <Tbody>
+                      {waktu.map(({ id, kuota }) => {
+                        const { sisaKuota = 0, totalKuota = 0 } = kuota as Kuota;
+                        return (
+                          <Tr key={id}>
+                            <Td>{format(parse(id, 'k:m:s', new Date()), 'p')}</Td>
+                            <Td color={sisaKuota > 0 ? 'green' : 'red'} fontWeight={800}>
+                              {sisaKuota}
+                            </Td>
+                            <Td>{totalKuota}</Td>
+                          </Tr>
+                        );
+                      })}
+                    </Tbody>
+                  </Table>
+                </Box>
+              ))}
+            </Stack>
+          </DrawerBody>
+        </DrawerContent>
+      </Drawer>
```

Comment:
Can we split this drawer into a separate component? `modules/vax/VaxLocationDetailDrawer.tsx`

Reply:
Sure, I'll do it

Reply:
VaxLocationDetailDrawer.tsx added. kindly check the new commit https://github.com/tibudiyanto/jakarta-vax-availability/pull/79/commits/79a3dee10472a35c780a4d761fec646ae7fa33f9